tests/e2e/specs/modules/search-console/dashboard-date-range.test.js

```diff
@@ -52,7 +61,17 @@ describe( 'date range filtering on dashboard views', () => {
 
 		await page.setRequestInterception( true );
 		useRequestInterception( ( request ) => {
-			if ( request.url().match( 'google-site-kit/v1/data/' ) ) {
+			const url = request.url();
+			// Widget API requests. As mentioned above, these can be refactored
+			// to use the mock response utility as part of #2586.
```

Comment:
```suggestion
			// to use the mock response utility as part of https://github.com/google/site-kit-wp/issues/2586.
```
Let's link to the issue rather than just using the number 😄